### PR TITLE
added responsiveness to the site

### PR DESCRIPTION
**Pull request by ritik307**
## What is the change?
Added responsiveness to the mobile and tablet view of the website

## Related issue?
close: #739 

## How was it tested?
1. Click F12
2.  Change the mode to mobile or tablet.
3. The website will NOT have an unnecessary margin on the left side and the content of the website will NOT look distorted.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:

https://user-images.githubusercontent.com/22374829/117200760-7a04b680-ae09-11eb-86b7-830d42ccce12.mp4

### CODE REVIEW

**Comment by ritik307** [comment]
@AnanyaNegi @Pranav016 Plz review

File: public/css/styles.css [hunk 1]
```diff
@@ -266,7 +260,7 @@ a::after {
 
 .post-preview {
 	background-color: rgba(229, 231, 235, 0.5);
-	margin: 0.8rem;
+	/* margin: 0.8rem; */
```
Comment (Pranav016) [inline public/css/styles.css:263]: remove this line of code

File: views/partials/header.ejs [hunk 2]
```diff
@@ -13,7 +13,10 @@
     <link rel="icon" href="/images/favicon.ico" />
     <script src="https://kit.fontawesome.com/7a2242c499.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
-    <nav id="nav-id" class="navbar navbar-expand-md navbar-default darkmode--activated sticky-top">
+    
+
+    <body>
+      <nav id="nav-id" class="navbar navbar-expand-lg navbar-default darkmode--activated sticky-top">
```
Comment (Pranav016) [inline views/partials/header.ejs:19]: shouldn't the nav be in the header instead of body

**Review by Pranav016 — CHANGES_REQUESTED** [review]

File: views/partials/header.ejs (hunk 2 above)
Comment (ritik307) [reply views/partials/header.ejs:19]: I don't think so by the convention _The head element contains metadata—information that describes the document itself or associates it with related resources, such as scripts and style sheets. and the rest should belong to the body._

[Reference Link](https://stackoverflow.com/questions/31768460/should-i-place-my-navbar-before-or-after-the-body-tag-when-customizing-a-wp)

**Comment by ritik307** [comment]
@Pranav016 The issue is of level 3 and you have added a wrong level tag plz fix it.

**Pull request merged**